ui/de-lib/src/main/java/org/iplantc/de/client/DEClientConstants.properties

```diff
@@ -2,4 +2,7 @@ fileUploadServlet = /de/secured/fileUpload
 fileDownloadServlet = /de/secured/fileDownload
 logoutUrl = j_spring_security_logout
 deHelpFile = https://pods.iplantcollaborative.org/wiki/display/DEmanual/
+notificationWS = /de/websocket/notifications
+sysmessageWS = /de/websocket/system-messages
+wsProtocol = ws://
```

Comment:
In prod, will this be `wss://` since we'd be moving things to SSL there? I'd guess it should be, though maybe that's a task for another patch.